packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx

```diff
@@ -27,7 +27,7 @@ Bootstrap offers sizing classes with the suffixes bellow. They allow you to set
 
 #### Example
 
-<Canvas of={SizingStories.SizesPercent} />
+<Canvas sourceState="shown" of={SizingStories.SizesPercent} />
```

Comment:
I would recommend expanding the source only on the first canvas on components, but leave foundation pages as they are.